src/globals.cpp

```diff
@@ -289,3 +289,43 @@ CStatHistory<uint64_t> nBlockValidationTime("blockValidationTime", STAT_OP_MAX |
 CThinBlockData thindata; // Singleton class
 
 uint256 bitcoinCashForkBlockHash = uint256S("000000000000000000651ef99cb9fcbe0dadde1d424bd9f15ff20136191a5eec");
+
+#ifdef ENABLE_MUTRACE
+class PrintSomePointers
```

Comment:
sloppy class name?

Reply:
Do you want it to be CPrintSomePointers?  This class is private to this .cpp file and only used to create a global variable that prints something out.  Since is not really a class that people should use (not part of the API) I didn't worry abt it.  But I'll change if you think it matters.